jsonschema/validators.py

```diff
@@ -20,6 +20,12 @@
     ValidationError,
 )
 
+from jsonschema._format import (
+    draft3_format_checker,
+    draft4_format_checker,
+    draft6_format_checker,
```

Comment:
Draft 7 is on master too, so would be great if you just add one more line here and below to do the same as what you've done for draft 7.